biaspaltsau_a/14.Ansible.Workshop/roles/db/tasks/main.yaml

```diff
@@ -0,0 +1,27 @@
+---
+- name: MySQL. Install and setup
+  template:
+    src: my.cnf.j2
+    dest: ~/.my.cnf
+    owner: root
+    mode: 0600
+  notify:
+    - mysql restart
+
+- meta: flush_handlers
+
+- name: Configuring encoding DB
+  mysql_db:
+    name: "{{ app_db_name }}"
+    encoding: utf8
+
+- name: Configuring user priviliges
+  mysql_user:
+    name: User name "{{ app_db_user }}"
+    password: "{{ app_db_pass }}"
+    priv: "{{ app_db_name }}.*:ALL"
+
+- name: Config database
```

Comment:
This is task which belongs to application role, please move

Reply:
corrected